corehq/apps/geospatial/templates/geospatial/partials/index_alert.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+
+{% if task_status.status == 'ERROR' %}
+<div class="alert alert-danger">
+  <p>
+    {% if task_status.error_slug == 'TOO_MANY_CASES' %}
+      {% blocktrans %}
+        This domain contains too many cases and so they will not be made available
+        for use by this feature. Please reach out to support.
+      {% endblocktrans %}
+    {% elif task_status.error_slug == 'CELERY' %}
+      {% blocktrans %}
+        Oops! Something went wrong while attempting to make cases ready for use
+        by this feature. Please reach out to support.
+      {% endblocktrans %}
+    {% endif %}
+  </p>
+</div>
+{% elif task_status.status == 'ACTIVE' %}
+<div class="alert alert-info">
+  <p>
+    {% blocktrans %}
+      Cases are being made ready for use by this feature. Please be patient.
```

Comment:
nit: I'd suggest using "microplanning" instead of "this feature". I don't think users would know what feature you are referring to

Reply:
Please see [this](https://github.com/dimagi/commcare-hq/pull/35126#discussion_r1851773456) comment where I've suggested a possible refactor of the paragraph.

Reply:
Cool, I'd suggest doing the same change to other messages as well.